src/Query/Query.php

```diff
@@ -453,9 +453,7 @@ protected function queryScalar($selectExpression)
 
             try {
                 $command = $this->createCommand();
-            } catch (\Exception $e) {
-                /** throw it later */
-            } catch (Throwable $e) {
+            } catch (\Exception | Throwable $e) {
```

Comment:
```suggestion
            } catch (\Throwable $e) {
```

No need to catch \Exception separately. It is throwable as well.

Reply:
Done.